change/@microsoft-teams-js-8835237b-4a82-4082-b65a-5db37e32c03e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Removed the @beta tag from `pages`, `app`, and `authentication`. Added @beta tags to `chat`, `dialog`, and `profile` capabilities.",
```

Comment:
>Added @beta tags [](http://example.com/codeflow?start=79&length=16)

Maybe this should be "cascaded existing @beta tags in..." to make it clear that we aren't undoing previously solidified capabilities.